gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/PatientMapper.java

```diff
@@ -0,0 +1,59 @@
+package uk.nhs.adaptors.pss.translator.mapper;
+
+import org.hl7.fhir.dstu3.model.Identifier;
+import org.hl7.fhir.dstu3.model.Meta;
+import org.hl7.fhir.dstu3.model.Organization;
+import org.hl7.fhir.dstu3.model.Patient;
+import org.hl7.fhir.dstu3.model.Reference;
+import org.hl7.v3.RCMRMT030101UK04Patient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import lombok.RequiredArgsConstructor;
+import uk.nhs.adaptors.pss.translator.service.FhirIdGeneratorService;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class PatientMapper {
+
+    private final FhirIdGeneratorService idGenerator;
+
+    private static final String NHS_NUMBER_SYSTEM_URL = "https://fhir.nhs.uk/Id/nhs-number";
+    private static final String META_PROFILE_URL = "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Patient-1";
+    private static final String META_VERSION_ID = "1521806400000";
+
+    public Patient mapToPatient(RCMRMT030101UK04Patient patient) {
+        var identifier = createIdentifier(patient.getId().getExtension());
+        return createPatient(identifier, null);
```

Comment:
Instead of passing null to createPatient method, you could move setting managingOrganisation to the `mapToPatient(RCMRMT030101UK04Patient patient, Organization organization)` method. For example call createPatient to get a patient, then set managingOrganisation and return the patient object.